scripts/workers/code_upload_worker_utils/monitor_submission.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+echo "Monitoring submission..."
+time_elapsed=0
+time_delta=3600
```

Comment:
@savish28  Can we have this as an environment variable or part of the configMap? Avg runtime may vary depending on the type of challenge 
What do you think @Ram81?

Reply:
@KhalidRmb, yeah thought of that looked a good idea but the thing is it would make sense that if we keep it in challenge model somewhere.

Reply:
Right.

Reply:
Time delta is just for our time saving of the job running and doesn't directly deal with challenge i think. For eg if the timelimit is less that timedelta than it won't cost us much.

So that was the idea. But if you suggest i can put it in the challenge config.

Reply:
It matters if say the submissions are processed in much less than an hour or just over a whole number of hours, and we spend a not so insignificant amount of time idle.

Submission time limit is static right? Isn't that set by default to 1 day?

Even if you put time_delta in the config, it should be taken from the challenge model only right?...you could set it as an environment variable like the submission time limit as well.

Reply:
Yeah, in this case where submission are processed early on than time_delta and time limit is huge.
It makes sense to pass it as env variable. I would do that change.

I was thinking for eg If i am the host of the challenge i would not care at what interval the submission are monitored i guess. 😅. 

Would add as an env variable and add a field in challenge fot both time_delta and time_limit

Reply:
The challenge host need not set it per se. You can leave default values as the ones right now, but the point is that it's configurable. 

I agree it's not absolutely necessary right now, just a neat way to do things if we decide for whatever reason in the future to have different kind of challenges have different time limits.

Thoughts @Ram81

Reply:
@KhalidRmb added time_delta it in ENV, would add both fields in challenge model as well.

Reply:
Env change is fine. Wait until Ram confirms for the challenge model change. Thanks!

Reply:
In today's scrum ram and Rishabh agreed on adding submission_time_limit, so let's add that now and would add time_delta later. Cool?

Reply:
Cool